### ISSUE TITLE
upgrade schnorrkel to 0.10

### ISSUE THREAD

**Issue by gilescope** [paritytech/substrate#10048]
we are a few versions behind and this gets us to rand 0.8.

linked to upgrade substrate-bip39 crate.

**Comment by nazar-pc** [comment]
I saw #10025, is there a good reason schnorrkel wasn't updated yet?

**Comment by nazar-pc** [comment]
I see there is a verification bench in `schnorrkel`, did you use that to compare 0.9 to 0.10? Maybe other benches like signing should be added as well?

**Comment by burdges** [comment]
Yes likely so, but verification is by far the most important benchmark.

I donno how big the substrate regression was either.
